packages/scoutgame/src/farcaster/connectFarcasterAccount.ts

```diff
@@ -26,4 +29,8 @@ export async function connectFarcasterAccount({ fid, userId }: { fid: number; us
     where: { id: userId },
     data: { farcasterId: fid, farcasterName: profile.username }
   });
+
+  await completeQuests(userId, ['link-farcaster-account']);
```

Comment:
the bug was that completeQuests would be called even if we returning a 'connected user'